src/hotspot/share/compiler/compileBroker.cpp

```diff
@@ -2777,6 +2770,13 @@ void CompileBroker::print_times(bool per_compiler, bool aggregate) {
     tty->cr();
     comp->print_timers();
   }
+#if INCLUDE_JVMCI
+  if (EnableJVMCI) {
```

Comment:
The `!UseJVMCICompiler` condition has been removed as it's possible for both CompilerBroker and hosted compilations to be made in a single VM execution.